hivemind_core/protocol.py

```
@@ -231,6 +210,16 @@
         except:
             LOG.exception("error on connect callback")
 
+        try:  # let the binary protocol know about it
+            self.binary_data_protocol.callbacks.on_connect(client)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)